tests/test_json_contract_loader.py

```
@@ -0,0 +1,45 @@
+"""Tests for the JSON contract loader utility."""
+from pathlib import Path
+
+from json_contract_loader import JSONContractLoader
+
+
+def test_load_report_assembly_contracts(tmp_path: Path) -> None:
```

Ruff (0.14.1)

Unused function argument: `tmp_path`

(ARG001)